x-pack/plugins/cases/public/components/case_view/index.test.tsx

```diff
@@ -160,6 +163,10 @@ describe('CaseView ', () => {
       pushCaseToExternalService,
     }));
     useConnectorsMock.mockImplementation(() => ({ connectors: connectorsMock, loading: false }));
+    useKibanaMock().services.triggersActionsUi.actionTypeRegistry.get = jest.fn().mockReturnValue({
```

Comment:
Will the tests throughout this PR fail if `get` returns `undefined`? I'm just wondering if it's sufficient to `jest.mock('../../common/lib/kibana')`

Seems like we do this in some places: https://github.com/elastic/kibana/pull/101279/files#diff-94a9922e9734d8791131f05c1124fd6197bdceb756e6bc2770191639b905e169R85

Reply:
I see what you mean! It is not sufficient because of the way the mock is being created (`triggersActionsUiMock.createStart()` in `x-pack/plugins/cases/public/common/lib/kibana/kibana_react.mock.ts`). The `actionTypeRegistry` is not mocked so when the flyout access the `actionTypeRegistry` to get the connector type you will get an error that the type is not registered.